ostree/ostree_selinux.go

```diff
@@ -0,0 +1,74 @@
+// +build !containers_image_ostree_stub,selinux
+
+package ostree
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"unsafe"
+
+	selinux "github.com/opencontainers/selinux/go-selinux"
+	"github.com/pkg/errors"
+)
+
+// #cgo pkg-config: glib-2.0 libselinux
+// #include <glib.h>
+// #include <stdlib.h>
+// #include <selinux/selinux.h>
+// #include <selinux/label.h>
+import "C"
+
+type mandatoryAccessControl struct {
+	handler *C.struct_selabel_handle
+}
+
+func (m mandatoryAccessControl) Open() error {
+	var selinuxHnd *C.struct_selabel_handle
+
+	if os.Getuid() == 0 && selinux.GetEnabled() {
+		selinuxHnd, err := C.selabel_open(C.SELABEL_CTX_FILE, nil, 0)
```

Comment:
This SELinux handle will not reset the one below.
Remove ':' from :=

Reply:
@rhatdan thanks for the prompt review!

The error is not in the assignment, but in [line 29](https://github.com/containers/image/pull/562#discussion-diff-248706597R29), which isn't needed.

Removing it.